common/src/main/java/org/opentestsystem/rdw/admin/common/security/PermissionScope.java

```diff
@@ -2,13 +2,14 @@
 
 import com.google.common.collect.ImmutableSet;
 
+import java.io.Serializable;
 import java.util.Collection;
 import java.util.Set;
 
 /**
  * Used to limit database query results
  */
-public class PermissionScope {
+public class PermissionScope implements Serializable {
```

Comment:
The Serializable annotations caught me off-guard.  I didn't realize they were required until I ran the app against a local redis service.

Reply:
These are required now with the new redis serialization?

Reply:
Yeah, because redis by default uses a JdkSerializationRedisSerializer to serialize session information.  That serializer then uses an ObjectOutputStream/ObjectInputStream to serialize/deserialize session information using Java serialization.  So only Serializable thingies are allowed to exist in the session store.

Reply:
Changing it to use a Jackson/JSON serializer would require some custom serializers/deserializers to handle the SAML XML payloads.

Reply:
Ok I remember that now. I thought we changed it to use JSON for some reason but i remember now that I couldn't get that working.